docs/bokeh/docserver.py

```diff
@@ -55,9 +55,9 @@ def root():
     return redirect("en/latest/index.html")
 
 
-@app.route("/switcher.json")
+@app.route("/en/switcher.json")
```

Comment:
Combining `en/latest/` and `../switcher.json` results in `/en/switcher.json` for `route`.